tests/conftest.py

```diff
@@ -12,7 +13,7 @@ def shared_setup(module_isolation):
 
 @pytest.fixture
 def safe():
-    return GreatApeSafe(registry.eth.badger_wallets.ops_multisig)
+    return GreatApeSafe(accounts[9].address)
```

Comment:
any specific reason for `accounts[9]`? might `accounts.add()` not be better? or will that unnecessarily generate a new safe every time when the scope gets reset?

Reply:
originally had `accounts.add()`, however `test_badger_pool` and `test_convex_claim`
fail as opposed to using `accounts[x]`. still investigating it, but not sure why it would make enough of a difference for these tests to fail